Signal/src/view controllers/CallViewController.swift

```diff
@@ -472,7 +473,28 @@ class CallViewController: UIViewController, CallDelegate {
         case .answering:
             return NSLocalizedString("IN_CALL_SECURING", comment: "Call setup status label")
         case .connected:
-            return NSLocalizedString("IN_CALL_TALKING", comment: "Call setup status label")
+            if let call = self.call {
+                let callDuration = call.connectionDuration()
+                let callDurationDate = Date(timeIntervalSinceReferenceDate:callDuration)
+                let dateFormatter = DateFormatter()
```

Comment:
I've moved this `DateFormatter` to a lazily-populated class property. Formatting feels like a view concern, so I'd rather it didn't leak into the model.  

Thanks @pouriaalmassi @michaelkirk